docs/modules/ROOT/pages/includes/attributes.adoc

```diff
@@ -1,3 +1,4 @@
+:maven-version: 3.9.2
```

Comment:
ℹ️ I wasn't exactly sure which was the minimal maven version. If it's not 3.9.2 I'll of course change this line.

Reply:
That's okay, it usually should be the same as defined here: https://github.com/quarkusio/quarkus/blob/main/build-parent/pom.xml#L67